Graham_scan_Algorithm/main.cpp

```diff
@@ -0,0 +1,108 @@
+/*The convex hull is the minimum closed area which can cover all given data points.
```

Comment:
@Anshuvats Since the PR is for MO's Algorithm, Remove this file

Reply:
To squash commit see[ this](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git)

Reply:
okay sir,i will try once again as soon as possible